### ISSUE TITLE
Option to 'set Files as Default File Explorer' doesn't get undo/disabled

### ISSUE THREAD

**Issue by gdmcoder** [files-community/Files#7221]
So, i intentionally enabled the option to 'set Files as Default File Explorer' but now, i want to go back to default Windows File Explorer so, first i tried to disable that option 'set Files as Default File Explorer' but it wasn't going back 

![k2wML36xQt](https://user-images.githubusercontent.com/95571677/144745910-15e74b31-f76f-4b53-83b2-7c8f01ccf20d.gif)

so, then i uninstall the app but after doing that, when i pressed windows + E key to open the file Explorer, i got an error 'Application not Found'. 

![explorer_v20Iw41zey](https://user-images.githubusercontent.com/95571677/144746101-dfeed2a0-d5dc-4584-a099-6e0dc3f63378.png)

Then i looked for the default app settings in windows but couldn't do anything there

Now, i have to use The Files app which i don't want to so, please help

**Comment by gave92** [comment]
To restore windows explorer, download [this zip](https://raw.githubusercontent.com/files-community/files-community.github.io/main/data/SetFilesAsDefault.zip) and run UnsetFilesAsDefault.reg.

**Comment by gdmcoder** [comment]
Ohh Thanks bro👍😄

**Comment by ZaLiTHkA** [comment]
> To restore windows explorer, download [this zip](https://raw.githubusercontent.com/files-community/files-community.github.io/main/data/SetFilesAsDefault.zip) and run UnsetFilesAsDefault.reg.

this just saved me as well.. glad I searched here for more info.

tl;dr: Kasperksy freaked out when I tried to make Files my default file manager, and then I couldn't launch Files again to uncheck that option. it won't even let me install it again now... I needed the registry change from `UnsetFilesAsDefault.reg` to restore my <kbd>Win</kbd>+<kbd>e</kbd> keyboard shortcut.

is there perhaps somewhere this could be made a bit more visible to Files app users..? I don't see any sort of FAQ or Wiki or "common issues" info on either the main website or this project's docs that fits this role.
